src/components/game/GameCard.js

```diff
@@ -101,17 +109,14 @@ const CardBody = ({ article, nextButton }) => {
         onChange={changeHandler("reuters")}
       />
       <br />
+      <p>Score: {score}</p>
       <div className="submit-button">
         <button onClick={submitHandler}>Submit</button>
       </div>
       <button onClick={nextButton}>Next</button>
-      {isGuessCorrect === "correct" ? (
-        <CongratsCard article={article} />
-      ) : isGuessCorrect === "incorrect" ? (
-        <p>Incorrect, guess again</p>
-      ) : (
-        <p> Guess now! </p>
-      )}
+
+      {isGuessCorrect === "correct" && <CongratsCard article={article} />}
+      {isGuessCorrect === "incorrect" && <p>Incorrect, guess again!</p>}
```

Comment:
if `isGuessCorrect` is boolean you could just do 
`{isGuessCorrect && ...`
`{!isGuessCorrect && ...`

or ternary
```
{ isGuessCorrect ? <CongratsCard ... /> : <p></p> }
```